drivers/eaton-pdu-marlin-mib.c

```diff
@@ -41,176 +41,176 @@
-	{ 28, "nemaL630", NULL, NULL },
-	{ 29, "nemaL715", NULL, NULL },
-	{ 30, "rf203p277", NULL, NULL },
-	{ 0, NULL, NULL, NULL }
+	info_lkp_default(0, "unknown"),
+	info_lkp_default(1, "iecC13"),
+	info_lkp_default(2, "iecC19"),
+	info_lkp_default(10, "uk"),
+	info_lkp_default(11, "french"),
+	info_lkp_default(12, "schuko"),
+	info_lkp_default(20, "nema515"),
+	info_lkp_default(21, "nema51520"),
+	info_lkp_default(22, "nema520"),
+	info_lkp_default(23, "nemaL520"),
+	info_lkp_default(24, "nemaL530"),
+	info_lkp_default(25, "nema615"),
+	info_lkp_default(26, "nema620"),
+	info_lkp_default(27, "nemaL620"),
+	info_lkp_default(28, "nemaL630"),
+	info_lkp_default(29, "nemaL715"),
+	info_lkp_default(30, "rf203p277"),
+	info_lkp_sentinel
 };
 
 static info_lkp_t marlin_ambient_presence_info[] = {
-	{ -1, "unknown", NULL, NULL },
-	{ 0, "no", NULL, NULL },  /* disconnected */
-	{ 1, "yes", NULL, NULL }, /* connected */
-	{ 0, NULL, NULL, NULL }
+	info_lkp_default(-1, "unknown"),
+	info_lkp_default(0, "no"),	/* disconnected */
+	info_lkp_default(1, "yes"),	/* connected */
+	info_lkp_sentinel
 };
 
 static info_lkp_t marlin_emp002_ambient_presence_info[] = {
-	{ 0, "unknown", NULL, NULL },
-	{ 2, "yes", NULL, NULL },     /* communicationOK */
-	{ 3, "no", NULL, NULL },      /* communicationLost */
-	{ 0, NULL, NULL, NULL }
+	info_lkp_default(0, "unknown"),
+	info_lkp_default(2, "yes"),	/* communicationOK */
+	info_lkp_default(3, "no"),	/* communicationLost */
+	info_lkp_sentinel
 };
 
 static info_lkp_t marlin_threshold_status_info[] = {
-	{ 0, "good", NULL, NULL },          /* No threshold triggered */
-	{ 1, "warning-low", NULL, NULL },   /* Warning low threshold triggered */
-	{ 2, "critical-low", NULL, NULL },  /* Critical low threshold triggered */
-	{ 3, "warning-high", NULL, NULL },  /* Warning high threshold triggered */
-	{ 4, "critical-high", NULL, NULL }, /* Critical high threshold triggered */
-	{ 0, NULL, NULL, NULL }
+	info_lkp_default(0, "good"),	/* No threshold triggered */
+	info_lkp_default(1, "warning-low"),	/* Warning low threshold triggered */
+	info_lkp_default(2, "critical-low"),	/* Critical low threshold triggered */
+	info_lkp_default(3, "warning-high"),	/* Warning high threshold triggered */
+	info_lkp_default(4, "critical-high"),	/* Critical high threshold triggered */
+	info_lkp_sentinel
 };
 
 static info_lkp_t marlin_threshold_frequency_status_info[] = {
-	{ 0, "good", NULL, NULL },          /* No threshold triggered */
-	{ 1, "out-of-range", NULL, NULL },  /* Frequency out of range triggered */
-	{ 0, NULL, NULL, NULL }
+	info_lkp_default(0, "good"),	/* No threshold triggered */
+	info_lkp_default(1, "out-of-range"),	/* Frequency out of range triggered */
+	info_lkp_sentinel
 };
 
 static info_lkp_t marlin_ambient_drycontacts_info[] = {
-	{ -1, "unknown", NULL, NULL },
-	{ 0, "opened", NULL, NULL },
-	{ 1, "closed", NULL, NULL },
-	{ 0, NULL, NULL, NULL }
+	info_lkp_default(-1, "unknown"),
+	info_lkp_default(0, "open"),
```

Comment:
Is it "open" or "opened" here?.. Gotta look up in git blame for different branches...

Reply:
Should be "opened" per d3b6ea4d89 - fixed in master in 2017, stale in FTY branch

Reply:
Earlier also missed the HPE PDU mapping, fixed with this PR now.